client-evaluation/README.md

```diff
@@ -0,0 +1,15 @@
+# Evaluating clients
+
+## We don't do rewrites.
+
+#### Are we able to maintain the project after it's finished
+
+We want to do that for limited time only. Amount of maintained projects is limited. Exception might be creating own product.
+
+#### Is the client willing to take over the project maintenance/development
+
+We're looking for clients willing to incorporate inhouse team. We want to talk with bussiness people.
+
+#### Does the client understand iterative development
```

Comment:
Iterative, evolutionary or continuous? Iterative presumes iterations which are only one tool to get from nothing to something… 😉